civiform/civiform
# Add styling to the address correction page with 0 suggested addresses

### Description

This PR only adds styling to the version of the address correction page that shows 0 suggested addresses. (Mocks are only available for this case right now.)

[View mocks](https://www.figma.com/design/RmhVJCLgAKWGENJLWWAtNS/CiviForm%3A-Components%2C-Styles?node-id=3705-15213&m=dev)

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)


#### User visible changes


### Instructions for manual testing

1. Have a program with an address question
1. Apply to that program
1. Go to a block with an address question
1. Answer the address question (see below)
1. Click "Next"
1. Observe the address correction page

* To receive 0 suggestions: answer anything like "123 fake st"
* To receive multiple suggestions: answer "Address In Area"
* [source](https://github.com/civiform/civiform/wiki/Mock-Web-Services#address-correction)

### Issue(s) this completes

#7684
